lage.config.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+module.exports = {
+  pipeline: {
+    build: ["^build"],
+    lint: ["build"],
+    'lint:fix': [],
+    test: ["build"],
+    format: [],
+    'format:verify': [],
+    clean: [],
+  },
+  cacheOptions: {
+    outputGlob: [
+      'js/**',
```

Comment:
I wish we could do this per-package, but package local lage or backfill config doesn't seem to work as far as I can tell

Reply:
Yeah, looks like we don't support using package-local backfill config